installer/test/InstallerFunctionalTests/FunctionalTests.cpp

```diff
@@ -25,49 +25,60 @@ namespace ProjectReunionInstallerTests
         TEST_METHOD(VerifyInstallerRuns)
         {
             auto result = RunInstaller();
-            Assert::AreEqual(result, S_OK);
+            Assert::AreEqual(S_OK, result);
```

Comment:
VSTest?

RECOMMNEDATION: Use TAEF.

DynamicDependencies' tests use VSTest and given that 1st hand experience I'm in the process of rewriting them to TAEF... per [Rewrite DynamicDependencies tests using TAEF #559](https://github.com/microsoft/ProjectReunion/issues/559)

If not in this PR is there an Issue for follow up?

Reply:
I had difficulty using TAEF earlier, added an issue for followup, #714